src/renderer/components/player-settings/player-settings.js

```diff
@@ -62,6 +62,14 @@ export default defineComponent({
       screenshotFolderPlaceholder: '',
       screenshotFilenameExample: '',
       screenshotDefaultPattern: '%Y%M%D-%H%N%S',
+      viewingModeValues: [
+        'default',
+        'theatre',
+        ...(process.env.IS_ELECTRON ? ['fullscreen'] : []),
+        'fullwindow',
+        ...(process.env.IS_ELECTRON ? ['pip'] : []),
+        'external_player'
+      ]
```

Comment:
External players are also an Electron only thing.
As half of the list is Electron only, it would probably be simpler to just do `process.env.IS_ELECTRON ? [electron stuff] : [outside of electron stuff]`